### ISSUE TITLE
Credential fails to verify.

### ISSUE THREAD

**Issue by bddap** [docknetwork/linked-biometrics-demo#4]
vc-js verification fails. The result, nothing happens. No drink is dispensed. No indication of failure is shown to the user. An error is printed to console though; something about an invalid key type.

```
...Error: Invalid key type. Key type must be "Ed25519VerificationKey2018". at o.assertVerificationMethod...
```

## To reproduce:

Go to https://biometrics-demo.dock.io/ , issue yourself a credential bundle via webcam. Next upload the credential, you should see nothing happen.

I'm happy to provide more info on reproducing the issue if needed.

**Comment by bddap** [comment]
@SamHellawell @soyoungdock :wave:

**Comment by bddap** [comment]
fixed by #5
